private/bufpkg/bufcheck/buflint/buflint_test.go

```diff
@@ -459,29 +460,22 @@ func TestRunPackageLowerSnakeCase(t *testing.T) {
 
 func TestRunPackageNoImportCycle(t *testing.T) {
 	t.Parallel()
-	testLintWithModifiers(
```

Comment:
There should be no diff here - this function was formatted the same as the rest of this file, and there's no change here with respect to this PR.